### PR TITLE
Document how to purge a Solr core

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,13 +156,20 @@ and assign the image styles created in step3 to each breakpoint.
...
- - https://drupal-starter.ddev.site:9201/\_search - list of all documents
+The starter kit comes out of the box with Solr. Pantheon's built-in, very limited Solr is useful for basic sites in English mostly.
+For sophisticated needs, we can spin up an index at [OpenSolr](https://opensolr.com/) for instance, especially if we need stopword handling or synonyms.
+
+### Index cleanup
+
+It can happen that an index is polluted and Search API cannot restore it using "Delete all indexed items". Then there's a Drush command of the integration module to reset the index, drop all data inside:
+
+```bash
+terminus remote:drush gizra-drupal-starter.qa search-api-pantheon:force-cleanup
```
Comment (amitaibu) [inline README.md:169]: `terminus`here is executed outside of ddev. Let's have it inside ddev

Comment (AronNovak) [inline README.md:169]:
```suggestion
ddev . terminus remote:drush gizra-drupal-starter.qa search-api-pantheon:force-cleanup
```

File: README.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ For sophisticated needs, we can spin up an index at [OpenSolr](https://opensolr.
 It can happen that an index is polluted and Search API cannot restore it using "Delete all indexed items". Then there's a Drush command of the integration module to reset the index, drop all data inside:
 
 ```bash
-terminus remote:drush gizra-drupal-starter.qa search-api-pantheon:force-cleanup
+ddev . terminus remote:drush gizra-drupal-starter.qa search-api-pantheon:force-cleanup
```
Comment (amitaibu) [inline README.md:169]: Is it enough just to do `ddev auth ssh`, and it should "just work"?

```suggestion
ddev exec terminus remote:drush gizra-drupal-starter.qa search-api-pantheon:force-cleanup
```

Comment (AronNovak) [reply README.md:169]: Yep, `ddev auth ssh` is generally needed here, but also the machine token.

**Pull request merged**